Rakefile

```diff
@@ -17,3 +17,160 @@ namespace :pkg do
   desc 'Generate package source gem'
   task :generate_source => :build
 end
+
+namespace :plugin do
+  desc 'See rake plugin:draft -- --help for more information.'
+  task :draft do |_t, args|
+    require 'optparse'
+    require 'fileutils'
+
+    options = {
+      name: 'TODO', path: '../.', author: 'TODO', email: 'TODO'
+    }
+    OptionParser.new do |opts|
+      opts.banner = "Create a draft for a plugin.
+Example: rake plugin:draft -- --name=my_plugin --author='John Doe'\n
+Usage: rake plugin:draft -- [options]"
+      opts.on('-nNAME', '--name=NAME', "Plugin's name, e.g. my_plugin") { |name| options[:name] = name }
```

Comment:
Having no separator between short flag and value seems very unusual to me. Could we have '=' to be consistent with `--name=NAME`?